frontend/src/components/Inquiries/EditInquiry.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  Button,
```

Comment:
This file seems to be a copy of AddInquiry. I'm not sure what it's for or how it is used.

Reply:
reafactored and added edit button

Reply:
I think you are misunderstanding my comment. the edit buttons needs to be on the list of questions next to each question. right now your code does not make sense to me. this code is for a modal component. how is this modal being opened ? from where?

Reply:
I am understanding your comment. In order to address it I need to be able to see list of questions in main, or combine the PRs. 
So far this PR is scaffolding.

Reply:
the question list has been feature complete for a week. it is visible in the app currently

Reply:
if you are running your app you should be able to add inquiries. then you see a list of them

Reply:
it probably is possible to use the same component for both editing and adding.  but you have to get the logic right

Reply:
@mcverter can you resolve this